core/src/main/scala/akka/persistence/cassandra/query/EventsByPersistenceIdStage.scala

```diff
@@ -69,39 +70,33 @@ import com.github.ghik.silencer.silent
       selectEventsByPersistenceIdQuery: PreparedStatement,
       selectSingleRowQuery: PreparedStatement,
       selectDeletedToQuery: PreparedStatement,
-      session: Session,
-      customConsistencyLevel: Option[ConsistencyLevel],
-      customRetryPolicy: Option[RetryPolicy]) {
+      session: CqlSession,
+      profile: String) {
```

Comment:
I keep wanting some way to validate the profile

Reply:
as in that we've set it? Or that it exists? We could add something in the CassandraSession to say a profile should be set on a statement

Reply:
I finally clicked into the `.conf` file. At a minimum, perhaps we should consider validating a read/write exist, and a profile value that might be needed for our use that may be different from the default? e.g. a situation like https://github.com/akka/akka-persistence-cassandra/pull/596/files#r354814000. Or a setting that would cause us to fail downstream if not set or set to a size/duration minimum for a vague example.

Say the `datastax-java-driver.basic.request.page-size = 5000`, if too large based on their data, would cause timeout here, would be log it at least now?

Reply:
> Say the datastax-java-driver.basic.request.page-size = 5000, if too large based on their data, would cause timeout here, would be log it at least now?

I am not sure I understand, how could that ever cause a timeout?

Reply:
> At a minimum, perhaps we should consider validating a read/write exist

As in check it exists in configuration? Sounds senisble

Reply:
Added in https://github.com/akka/akka-persistence-cassandra/pull/596/commits/89505be4915c9c4cfe433baba67c0e785aa1578c